util/http/http_client.cc

```diff
@@ -133,6 +133,10 @@ bool Client::IsConnected() const {
   return socket_ && socket_->IsOpen();
 }
 
+bool Client::IsClosed(const std::error_code& ec) const {
+  return socket_ && socket_->IsConnClosed(ec);
```

Comment:
`FiberSocketBase::IsConnClosed` is a static function and you do not need this wrapper to call it.
just use it directly where you need.